src/rustup-cli/common.rs

```diff
@@ -339,6 +339,17 @@ pub fn list_components(toolchain: &Toolchain<'_>) -> Result<()> {
     Ok(())
 }
 
+pub fn list_installed_components(toolchain: &Toolchain<'_>) -> Result<()> {
+    let mut t = term2::stdout();
+    for component in toolchain.list_components()? {
+        if component.installed {
+            let name = component.name;
+            let _ = writeln!(t, "{}", name);
```

Comment:
I'd prefer to use `?` rather than dropping the result on the floor here.

Reply:
Understood. Should I change it? The other code for regular listing of components does the same thing, I copied the style from there.

Reply:
Let's change them all, then, IMO.